.github/workflows/ci.yml

```diff
@@ -158,42 +137,25 @@ jobs:
           name: fluvio-sc-logs
           path: /usr/local/var/log/fluvio/flv_sc.log
 
-
-
   k8_cluster_test:
     name: Kubernetes cluster test
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
         rust: [stable]
-    
+
     steps:
       - uses: actions/checkout@v2
-      - name: Install Helm
-        run: dev-tools/ci-replace-helm.sh
-        env:
-          HELM_VERSION: v3.3.4
-          OS: ${{ matrix.os }}
-      - run: helm version
+      - name: Install Fluvio Local Cluster
+        uses: ./
```

Comment:
This seems to be the way to use a [composite action in it's own repo](https://github.com/dmesquita/github-actions-tutorial/blob/1d533f76934cde714f780a83a0092e88b8e087cd/.github/workflows/use-action-same-repo.yml#L10).